src/textual/widgets/_text_log.py

```diff
@@ -92,19 +97,26 @@ def write(
         width: int | None = None,
         expand: bool = False,
         shrink: bool = True,
+        scroll_end: bool | None = None,
     ) -> Self:
         """Write text or a rich renderable.
 
         Args:
             content: Rich renderable (or text).
-            width: Width to render or ``None`` to use optimal width.
-            expand: Enable expand to widget width, or ``False`` to use `width`.
+            width: Width to render or `None` to use optimal width.
+            expand: Enable expand to widget width, or `False` to use `width`.
             shrink: Enable shrinking of content to fit width.
+            scroll_end: Enable automatic scroll to end, or `None` to use `self.auto_scroll`.
 
         Returns:
             The `TextLog` instance.
         """
 
+        if scroll_end is None:
+            auto_scroll = self.auto_scroll
+        else:
+            auto_scroll = scroll_end
```

Comment:
I thought we liked conditional expressions?

How do you typically determine whether you'll use a conditional expression or an `if: ... else: ...`?

```suggestion
        auto_scroll = self.auto_scroll if scroll_end is None else scroll_end
```

Reply:
I think it is just a stylistic thing for the most part.